### DISCUSSION TITLE
Need help modifying and styling a pdf

### DISCUSSION THREAD

**Discussion by zsofiaj** [thegetty/quire#801] [Get Help]
We now have the pdf download working for our book, but I have a number of questions about styling our pdf. 

Is there an easy way to avoid bad breaks such the ones as in the first three screenshots? And the em tags are currently showing up for all italics that appear in the running footers (see fourth screenshot), but perhaps that is a known issue. 

I also have some superfluous contents pages in the pdf that I wish I could omit, but I asked @geealbers about that, and it sounded like that might not be possible. I've attached the full pdf here as well, for reference.  I'm a little mystified generally about how the contents pages output to the pdfs. I wish I could include just the main Contents page and the Catalogue page, and I would like the Catalogue page to appear as on the web version, with all of the subsections and individual catalogue entries showing:

https://italian-paintings-march2023-updates.netlify.app/catalogue/

I'm not sure why they do not appear in the pdf. I can cut unnecessary contents pages easily enough, but I can't easily adjust the ones that I do want to keep. Any suggestions would be greatly appreciated!

<img width="549" alt="Ital_Paint_bad_breaks_Screen Shot 2023-07-26 at 6 06 22 PM" src="https://github.com/thegetty/quire/assets/68071544/4a36a9c7-ce55-46ca-8e16-ad2613ee820b">

<img width="519" alt="Ital_Paint_bad_breaks_Screen Shot 2023-07-26 at 6 06 06 PM" src="https://github.com/thegetty/quire/assets/68071544/9ea99e93-6f2b-4498-9c58-19a104ca3a13">

<img width="461" alt="Ital_Paint_bad_breaks_Screen Shot 2023-07-26 at 6 05 48 PM" src="https://github.com/thegetty/quire/assets/68071544/cc2ec27f-8150-40b0-b1b8-a36f119de2e3">

<img width="1117" alt="Ital_Paint_running_footers_Screen Shot 2023-07-26 at 6 10 26 PM" src="https://github.com/thegetty/quire/assets/68071544/e2bab75b-9f45-4f6f-b396-0c048b159adb">

[Ital_Paintings_test03_reduced.pdf](https://github.com/thegetty/quire/files/12177128/Ital_Paintings_test03_reduced.pdf)

**Comment by Erin-Cecele** [comment]
Thanks @zsofiaj, while I don't have an ETA, we are working on adding CSS styling guidance for PDFs to the documentation because we feel it could benefit the wider Quire community. Thank you for your patience while we pull this information together.

**Comment by zsofiaj** [comment]
Thanks so much, @Erin-Cecele ! This will be so helpful. I had another pdf issue come up at the end of last week. It seems like it might be a bug, so I'll create a new post in the Issues section.

**Comment by Erin-Cecele** [comment]
Just to update you, we are currently drafting an update to the docs in [PR #692](https://github.com/thegetty/quire-docs/pull/692). You can follow along with our progress.

**Reply by zsofiaj** [reply]
Thank you, @Erin-Cecele ! I'll check this out.

**Reply by Erin-Cecele** [reply]
@zsofiaj, we are moving fast today. PR #692 is going to be merged any moment now. So you can check the [Output Your Project](https://quire.getty.edu/docs-v1/multiformat-output/#pdfprint-output) section of the documentation later today for some helpful instructions.

**Reply by zsofiaj** [reply]
@Erin-Cecele @geealbers  Thank you for adding all of those pdf styling tips to the Quire documentation! So helpful. I made a ton of little image adjustments throughout our Italian Paintings project, and the pdf is in much better shape now. Those two tall vertical images are fine as well. 

I have a few additional style questions. Is there a way to simply get a line of text to break to the next column? There are many instances of headers ending up on a line by themselves. See, for example, pages 11, 56, 90, 91, and 95. In some of these instances, an image follows the header. That isn't ideal anyway, but that is where all of the "is-pulled-right" images are showing up, since they are inserted in the md file above each paragraph in which they appear. I wish there were a way to place all of these after the paragraph. Perhaps that is something that could be written into Quire? Wouldn't it be best if in the pdf each image appeared after the associated paragraph, since the images aren't being inserted into the paragraph anyway? But there are some instances of a header stuck on its own that do not involve images. 

I'm not sure if my image attachments are working below this time. If not, I'll try attaching as screenshots in a post below. 

Please see also the page 97 image. I wish there were a way to keep all of that text in a single column. Is that impossible?

And there are several spots where I wish I could just insert a column break or a page break in the text itself, for example to get our references and notes sections to appear on the next page. See pages 115–16. 

Oh, and my example from pages 123–24 was one particularly ugly example in which the header appears on its own between two images. 

Grateful for any advice you might have for these! 

![Ital_Paintings_Aug-29_p11](https://github.com/thegetty/quire/assets/68071544/1a733ea2-a964-4a05-8311-04c25fa51c95)
![Ital_Paintings_Aug-29_p56](https://github.com/thegetty/quire/assets/68071544/1b32e116-dc1f-41fa-b8a3-1dc16993b193)
![Ital_Paintings_Aug-29_p90](https://github.com/thegetty/quire/assets/68071544/ac0c6af3-0ced-414d-8f60-688b22899ac3)
![Ital_Paintings_Aug-29_p91](https://github.com/thegetty/quire/assets/68071544/a1494dc6-e73d-4b28-9ee1-657ff62a25ea)
![Ital_Paintings_Aug-29_p95](https://github.com/thegetty/quire/assets/68071544/a096b0bf-8732-40f4-a768-0084937e1f3d)
![Ital_Paintings_Aug-29_p97](https://github.com/thegetty/quire/assets/68071544/5e02f1d3-86bc-40ff-af62-302750cf8b74)
![Ital_Paintings_Aug-29_p115](https://github.com/thegetty/quire/assets/68071544/87b95af1-ddb2-46b6-8a30-d3b7f8e08b89)
![Ital_Paintings_Aug-29_p116](https://github.com/thegetty/quire/assets/68071544/93ec0598-d504-4b80-84a5-3144f575a72d)
![Ital_Paintings_Aug-29_p123](https://github.com/thegetty/quire/assets/68071544/6e9e8348-c8de-42f6-a9ab-41da3602e51a)
![Ital_Paintings_Aug-29_p124](https://github.com/thegetty/quire/assets/68071544/daf1ffd1-0f41-4932-8169-f7270f8fa1c0)

**Comment by Erin-Cecele** [comment]
As for the hiding of certain pages in the PDF, is this helpful: https://quire.getty.edu/docs-v1/pages/#hideshow-pages?

**Reply by zsofiaj** [reply]
Thank you! I did try adjusting the output values earlier to hide some subsection index pages, and I hit some snags. But now I can't recall what did and did not work. I will try playing around with this on Monday, and hopefully I can get the pdf into better shape overall now that you have added more tips for using custom styles to refine the pdf layout. Will report back soon.

**Reply by zsofiaj** [reply]
I'm sorry to post twice on this same thread, but I'll reply about the "outputs" attributes here. I played around with hiding some of the subsection index pages in the pdf by excluding them from the pdf output, but it didn't work as I expected. In the pdf with file name ending in A, I made contents pages visible for all of the various levels: main TOC, Catalogue (which shows all of the subsections as well as each individual entry), Volume 1/2/3, and the individual schools. I would like to keep the main TOC and the Catalogue page as they appear in example A, but I do not want the Volume 1 page or the grid view of the Florentine School. When I removed "pdf" from the outputs of the Florentine School and Sienese School indexes, however, it also removed the full list from the Catalogue page (see example B below). Is it possible to make the full catalogue contents show up on the Catalogue page, but without showing the full contents again for each of the subsection indexes? Ideally, I would like to keep a section title page for Volume 1/2/3, and for each of the schools, but I would not like for a contents list or grid view to show on those pages. I hope that makes sense! Maybe this is not possible though. 

[Ital_Paintings_Aug-29_contentsA.pdf](https://github.com/thegetty/quire/files/12470341/Ital_Paintings_Aug-29_contentsA.pdf)
[Ital_Paintings_Aug-29_contentsB.pdf](https://github.com/thegetty/quire/files/12470342/Ital_Paintings_Aug-29_contentsB.pdf)
